files/ko/web/javascript/reference/errors/bad_return_or_yield/index.html

```diff
@@ -43,9 +64,14 @@ <h2 id="예제">예제</h2>
   }
 };</pre>
 
-<h2 id="또_다른_내용">또 다른 내용</h2>
+<h2 id="See_also">또 다른 내용</h2>
```

Comment:
```suggestion
<h2 id="See_also">같이 보기</h2>
```
See also는 가이드라인에서 `같이 보기`로 번역하는 것으로 정해져서 리뷰했습니다. 밑에는 참고 링크입니다.

https://github.com/mdn/translated-content/blob/main/docs/ko/translation-guide.md#%EC%9A%A9%EC%96%B4-%EC%A7%80%EC%B9%A8